examples/todo/src/__tests__/unit/controllers/todo.controller.unit.ts

```diff
@@ -1,161 +0,0 @@
-// Copyright IBM Corp. 2019. All Rights Reserved.
-// Node module: @loopback/example-todo
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
-
-import {Filter} from '@loopback/repository';
-import {
-  createStubInstance,
-  expect,
-  sinon,
-  StubbedInstanceWithSinonAccessor,
-} from '@loopback/testlab';
-import {TodoController} from '../../../controllers';
-import {Todo} from '../../../models/index';
-import {TodoRepository} from '../../../repositories';
-import {GeocoderService} from '../../../services';
-import {aLocation, givenTodo} from '../../helpers';
-
-describe('TodoController', () => {
```

Comment:
I see little value in writing unit tests for controllers contributed by 3rd-party packages. The controller code should have been already tested in that package (`@loopback/rest-crud` in this case).